src/NSwag.Commands/HostApplication.cs

```diff
@@ -140,7 +140,13 @@ void OnEntryPointExit(Exception exception)
             try
             {
                 // Get the IServiceProvider from the host
+#if NET6_0
```

Comment:
`NET6_0_OR_GEATER` or else it will be broken in NET 7 again

Reply:
Good idea. 
From my understanding, NSwag style has been code tailored to each dotnet version and I made it NET6 specific. DotNet has had many breaking changes in recent versions, so I do hope this works for NET7 as well. :-)